### DISCUSSION TITLE
Live Contest Logging: Direction and distance

### DISCUSSION THREAD

**Discussion by dg3feh** [magicbug/Cloudlog#3094] [Ideas / Feature Requests]
Hi!

in the normal logging mode are direction and distance between the two grid locators shown. Unfortunately this is not the case in contest logging mode. Could you please show that here too?

73 es thx, Holger DG3FEH

**Comment by magicbug** [comment]
Yes this is a reasonable suggestion, I'll get it tasked to be added

**Reply by dg3feh** [reply]
Hej!

Any news on that?

73, Holger
